sonar-helm-for-iac/converters/data-loading_test.go

```diff
@@ -8,39 +8,36 @@ import (
 
 func Test_store_multiple_sources_in_memory(t *testing.T) {
 	templateSources := &TemplateSources{
-		Name:        "test-project/templates/a.yaml",
-		RawTemplate: "apiVersion: v1",
-		fileNameToFileContent: map[string]string{
-			"values.yaml": "foo: bar",
-			"Chart.yaml":  "name: test-project",
-		},
+		Name: "test-project/templates/a.yaml",
+		files: filesFromStrings(map[string]string{
+			"test-project/templates/a.yaml": "apiVersion: v1",
+			"values.yaml":                   "foo: bar",
+			"Chart.yaml":                    "name: test-project",
+		}),
 	}
 
-	assert.Equal(t, 2, templateSources.NumAdditionalSources())
+	assert.Equal(t, 3, templateSources.NumSources())
+	assert.Equal(t, "apiVersion: v1", templateSources.TemplateFile())
 	assert.Equal(t, "foo: bar", templateSources.Values())
-	assert.Equal(t, "name: test-project", mo.TupleToResult(templateSources.SourceFile("Chart.yaml")).MustGet())
+	assert.Equal(t, "name: test-project", string(mo.TupleToResult(templateSources.SourceFile("Chart.yaml")).MustGet()))
 
 	data, _ := PrepareChartValues(templateSources)
 
 	assert.Contains(t, data, "Chart")
 }
 
 func Test_return_error_for_missing_file(t *testing.T) {
-	templateSources := NewTemplateSources(
-		"test-project/templates/a.yaml",
-		"apiVersion: v1",
-		map[string]string{"values.yaml": "foo: bar"})
+	templateSources := NewTemplateSources("test-project/templates/a.yaml", filesFromStrings(map[string]string{
+		"test-project/templates/a.yaml": "apiVersion: v1", "values.yaml": "foo: bar"}))
```

Comment:
```suggestion
		"test-project/templates/a.yaml": "apiVersion: v1", 
		"values.yaml": "foo: bar"}))
```

IMO we can have such formatting (one file per line) for better readability. It's compliant with `go fmt`

The same in next linex in this file